src/DGtal/helpers/Shortcuts.h

```diff
@@ -1651,6 +1653,67 @@ namespace DGtal
         else return surface->allVertices();
         return result;
       }
+      /// Outputs a digital surface as an OFF file (with its
+      /// topology).  Optionnaly you can specify the face colors (see
+      /// saveOBJ for a more advanced export).
+      ///
+      /// @tparam TDigitalSurfaceContainer any model of concepts::CDigitalSurfaceContainer
+      /// @tparam TCellEmbedder any type for maping Cell -> RealPoint.
+      ///
+      /// @param[in] digsurf the digital surface to output as an OBJ file
+      /// @param[in] embedder any map Cell->RealPoint
+      /// @param[in] face_colors either empty or a vector of size `trisurf.nbFaces` specifying the diffuse color for each face.
```

Comment:
Seeing the type face_colors should be face_color and is only one color. Or you have to update the code below to handle a vector of Colors.

Reply:
Thanks I forget ;)